src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -94,7 +98,6 @@ const RenderChildDashboard = props => {
           >
             <img
               className="child-dash-img"
-              // This icon will need to be changed to an inhouse icon, this is just imported as a placeholder //
```

Comment:
Should you leave this comment in for future students to eventually change this icon?